### ISSUE TITLE
How to deal with loss imbalance problems？

### ISSUE THREAD

**Issue by Liusandian** [miemie2013/Keras-SOLO#3]
When I train the SOLO algorithm, there will be a seesaw problem when the loss functioin training reaches a certain stage, that is to say, the class loss suddenly becomes larger, and the mask loss still maintains a downward trend. How should I avoid this situation? Should I stop training at this time?
Looking forward to your reply！ Thanks !

**Comment by miemie2013** [comment]
> When I train the SOLO algorithm, there will be a seesaw problem when the loss functioin training reaches a certain stage, that is to say, the class loss suddenly becomes larger, and the mask loss still maintains a downward trend. How should I avoid this situation? Should I stop training at this time?
> Looking forward to your reply！ Thanks !

Make the learning rate more lower.

**Comment by Liusandian** [comment]
> > When I train the SOLO algorithm, there will be a seesaw problem when the loss functioin training reaches a certain stage, that is to say, the class loss suddenly becomes larger, and the mask loss still maintains a downward trend. How should I avoid this situation? Should I stop training at this time?
> > Looking forward to your reply！ Thanks !
> 
> Make the learning rate more lower.

10 times decay or other rate decay is suitable?
